AutopanoPro.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">
+  <mime-type type="application/pano">
```

Comment:
this is an invalid mimetype.

Reply:
is it though? The point of this file is to register a new mime type.

Reply:
it is not IANA registered. There is a naming for non standard mimetypes.